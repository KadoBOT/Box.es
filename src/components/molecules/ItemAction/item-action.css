.item-action {
  height: 60px;
}

.item-action .fa {
  margin-right: 5px;
}
