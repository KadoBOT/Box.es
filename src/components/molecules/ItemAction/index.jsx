import React from 'react'

import { Button } from '../../atoms/'
import './item-action.css'

const ItemAction = () => (
  <div className="item-action">
    <Button>
      <i className="fa fa-comment" /> Buy
    </Button>
  </div>
)

export default ItemAction
