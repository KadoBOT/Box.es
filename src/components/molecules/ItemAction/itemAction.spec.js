import React from 'react'
import renderer from 'react-test-renderer';
import ItemAction from './index'

it('should render the ItemAction', () => {
    const wrapper = renderer.create(
        <ItemAction />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
});
