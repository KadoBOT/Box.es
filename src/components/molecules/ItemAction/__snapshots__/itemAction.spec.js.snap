exports[`test should render the ItemAction 1`] = `
<div
  className="item-action">
  <button
    className="ksLTTq">
    <i
      className="fa fa-comment" />
     Buy
  </button>
</div>
`;
